en/core-libraries/collections.rst

```diff
@@ -197,6 +197,20 @@ You can also optionally use a ``groupPath`` to group results based on a path::
         'a' => [1 => 'foo', 3 => 'baz'],
         'b' => [2 => 'bar']
     ];
+    
+Finally you can use ``closures`` to generate paths dynamically, for example when working with entities and dates (converted to ``Cake/Time`` instances by the ORM) you may want to group results by date::
```

Comment:
I think that "closures" need to be highlighted in some way, because is the focus of this example and the motivation of this PR. Can I highlight it without using backticks?